src/pages/About/Card.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro'
+
+const StyledCard = styled.a`
```

Comment:
use `ExternalLink` if possible

Reply:
yes this i gotta fix

Reply:
gonna do in separate pr bc i need the links still